firefox-ios/Client/FeatureFlags/LegacyFeatureFlagsManager.swift

```diff
@@ -66,6 +65,15 @@ class LegacyFeatureFlagsManager: HasNimbusFeatureFlags {
         }
     }
 
+    /// Allows us to override nimbus feature flags for a specific build using the debug menu
+    private func getNimbusOrDebugSetting(with feature: NimbusFlaggableFeature) -> Bool {
+        #if MOZ_CHANNEL_BETA || MOZ_CHANNEL_FENNEC
+        return feature.isDebugEnabled(using: nimbusFlags)
```

Comment:
@adudenamedruby since I want to override with checking the debug setting first, I decided to implement the method a bit different from what was originally suggested. Let me know if any concerns!